<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mes mots de passe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>363</x>
       <y>10</y>
       <width>20</width>
       <height>111</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Generate">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Générer un Mot de passe</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Password">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Validate">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>100</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider ce Mot de Passe</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Domaine:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Identifiant :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Mot de passe:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_Domain">
        <property name="placeholderText">
         <string>ATTENTION !! Utiliser un nom unique</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Id"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Pass"/>
      </item>
     </layout>
    </widget>
    <widget class="QSpinBox" name="spinBox_Length">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="value">
      <number>12</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Longueur :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Add">
     <property name="geometry">
      <rect>
       <x>118</x>
       <y>100</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>591</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>571</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <column>
      <property name="text">
       <string>Domaine</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MdP</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton_Open">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ouvrir la Base</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Remove">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>200</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string> ! Effacer !</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
